examples/v1beta1/pipeline/realtimeimageclassfication/pipeline/HPO/model.py

```diff
@@ -0,0 +1,205 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import tensorflow as tf
+import numpy as np
+import logging
+from datetime import datetime
+import os
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+from tensorflow.keras.models import Sequential
+import pathlib
+logger = tf.get_logger()
+logging.basicConfig(
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    datefmt="%Y-%m-%dT%H:%M:%SZ",
+    level=logging.INFO)
+print('Tensorflow-version: {0}'.format(tf.__version__))
+
+import os
+import argparse
+import json
+
+
+
+    
+
+# build model
+def build_model(learning_rate,data_augmentation,opt):
+    img_height = 180
+    img_width = 180
+    num_classes=5
+
+    if opt == "SGD":
+        optimizer = tf.keras.optimizers.SGD(learning_rate=learning_rate)
+    elif opt =="ADAM":
+        optimizer = tf.keras.optimizers.Adam(learning_rate=learning_rate),
+    else:
+        optimizer=tf.keras.optimizers.RMSprop(learning_rate=learning_rate)
```

Comment:
Should we document it? We will use this optimizer by default.

Reply:
I think this optimizer function is using by Katib.

Reply:
@gaocegege Do you mean document it somewhere in [README](https://github.com/kubeflow/katib/blob/9bebebfa951fc4097cd7090e9a756a641ac79d08/examples/v1beta1/pipeline/realtimeimageclassfication/README.md) ?

Reply:
Yeah. I think so.